content/riak/ts/1.6.0/using/querying/select/aggregate-functions.md

```diff
@@ -143,3 +146,29 @@ Returns `NULL` if no values were returned or all values were `NULL`.
 |-------------------|-------------|
 | sint64            | sint64 |
 | double            | double |
+
+### `PERCENTILE_DISC`, `PERCENTILE_CONT`
+
+Calculate a percentile, given as a value in the range [0..1], in a population of entries in the selection, with null values discarded.  The column argument must be of numeric type (`sint64`, `double` or `timestamp`).  Return type is the type of the column argument for `PERCENTILE_DISC`, and `double` for `PERCENTILE_CONT`.
```

Comment:
Unless the value can be the integer 0 or 1, I would suggest expressing the range as `[0.0..1.0]`.